com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py

```diff
@@ -0,0 +1,2 @@
+class ViewNotFoundError(Exception):
+    """Raised when the SWS view element cannot be located."""
```

Comment:
Both. When using the SWS rest API to find a view, we iterate through the list of views found at
```
https://streams-console-c6d1.ng.bluemix.net/streams/rest/instances/{instance-id}/views/
```
If we can't find a SWS view element corresponding to the view name given when this topology is built, this exception would be thrown.